Testing/functional/tests/cypress/integration/e2e/timeline/dependent.js

```diff
@@ -0,0 +1,254 @@
+const { AuthMethod, Dataset } = require("../../../support/constants");
+
+const authorizedDependentHdid = "162346565465464564565463257";
+const unauthorizedDependentHdid = "BNV554213556";
+
+const homePath = "/home";
+const unauthorizedPath = "/unauthorized";
+
+function assertDatasetPresence(dataset, expected = true) {
```

Comment:
Should the tests using this function moved to ui tests suite?

Reply:
I thought about that, but having them in e2e ensures we check actual data is coming back when it's enabled.  There wouldn't be much time savings from using fixtures for the non-enabled checks because they aren't querying the back-end.